packages/@ourworldindata/grapher/src/dataTable/DataTable.tsx

```diff
@@ -776,35 +946,59 @@ function ColumnHeader(props: {
                 firstSubdimension: subdimensionType === "start",
                 endSubdimension: subdimensionType === "end",
                 lastSubdimension,
+                deltaColumn: isDeltaColumn(subdimensionType),
             })}
             rowSpan={props.rowSpan ?? 1}
             colSpan={props.colSpan ?? 1}
             onClick={props.onClick}
+            style={{ minWidth: props.minWidth }}
         >
-            <div
-                className={classnames({
-                    deltaColumn: isDeltaColumn(subdimensionType),
-                })}
-            >
-                <span>{props.headerText}</span>
-                {sortable && (
-                    <SortIcon
-                        type={props.dataType}
-                        isActiveIcon={sortedCol}
-                        order={
-                            sortedCol
-                                ? props.sortOrder
-                                : colType === "entity"
-                                ? SortOrder.asc
-                                : SortOrder.desc
-                        }
-                    />
-                )}
-            </div>
+            <CellContent maxWidth={props.contentMaxWidth}>
+                <div className="content">
+                    {!isEntityColumn && sortIcon}
+                    <span>{props.headerText}</span>
+                    {isEntityColumn && sortIcon}
+                </div>
+            </CellContent>
         </th>
     )
 }
 
+function CellContent(props: {
+    maxWidth?: number
+    children?: React.ReactNode
+}): JSX.Element {
+    if (!props.maxWidth) return <>{props.children}</>
+    return <div style={{ maxWidth: props.maxWidth }}>{props.children}</div>
+}
+
+function SortIcon(props: {
+    isActiveIcon?: boolean
+    order: SortOrder
+}): JSX.Element {
+    const isActiveIcon = props.isActiveIcon ?? false
+    const activeIcon =
+        props.order === SortOrder.desc ? faArrowUpLong : faArrowDownLong
+
+    return (
+        <span
+            className={classnames({ "sort-icon": true, active: isActiveIcon })}
+        >
+            {isActiveIcon ? (
+                <FontAwesomeIcon icon={activeIcon} />
+            ) : (
+                <span style={{ display: "inline-block", width: "max-content" }}>
+                    <FontAwesomeIcon icon={faArrowUpLong} />
+                    <FontAwesomeIcon
+                        icon={faArrowDownLong}
+                        style={{ marginLeft: "-4px" }}
+                    />
+                </span>
```

Comment:
Note for the reviewers: The "↑↓" icon is not part of the free collection of FontAwesome. That's why I'm faking it here.